.goreleaser.yml

```diff
@@ -70,3 +70,7 @@ release:
     - glob: ./deploy/manifests/nginx-plus-gateway.yaml
     - glob: ./deploy/manifests/nginx-gateway-experimental.yaml
     - glob: ./deploy/manifests/nginx-plus-gateway-experimental.yaml
+    - glob: ./deploy/manifests/openshift/openshift-nginx-gateway.yaml
```

Comment:
I would vote against adding more manifest files in general, but especially adding them to the release. I think we should push people to use Helm.

Reply:
I would also be an advocate to push people to use Helm, and if people request it we could always add it in. Also would be helpful to have insights to see if people are even installing through manifests.